### PR TITLE
Fixed typo in install.md file

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
## Styles and docs

- [ ] I have read the [contributing guide](https://github.com/hughrun/ephemetoot/blob/master/docs/contributing.md)
- [ ] I have run `black` on my code
- [ ] My tests are listed in alphabetical order

## Tests

- [ ] I have added tests for this code in `tests/test_ephemetoot.py`
- [ ] I would like assistance to write the required tests
- [ ] I don't know how to write test and would like someone to write them for me

# What this PR does

Changes in this pull request
This PR will be correcting the typo in install.md file in docs folder.

# Related issues

Resolves #
Fixes #

### CODE REVIEW

**Comment by 96RadhikaJadhav** [comment]
Hey @hughrun Why the checks are failed?

**Review by hughrun — APPROVED** [review]
Change looks good.

**Comment by hughrun** [comment]
@96RadhikaJadhav Checks look ok to me - there's no running code for it to be checking, and I see "0 checks". Sometimes GitHub does weird things if services are down though, so you may have seen something else, or possibly you saw a block because changes in this repo require formal approval before they can be merged (usually both are done by me, but I set that up to force myself to double check before I merge my own commits).

Thanks for contributing 😀

**Pull request merged**
